docs/tutorials/qaoa.ipynb

```diff
@@ -1,24 +1,16 @@
 {
```

Comment:
<p>nittiest of nit nits: extra space after better when you removed the extra better</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5200/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>